packages/blockchain-wallet-v4-frontend/src/scenes/Login/index.tsx

```diff
@@ -173,13 +173,33 @@ class Login extends PureComponent<InjectedFormProps<{}, Props> & Props, StatePro
               case LoginSteps.INSTITUTIONAL_PORTAL:
                 return <InstitutionalPortal {...loginProps} />
               case LoginSteps.ENTER_PASSWORD_EXCHANGE:
-                return <EnterPasswordExchange {...loginProps} />
+                return (
+                  <>
+                    {!loginProps.isMobileViewLogin && <UrlNoticeBar />}
```

Comment:
Could you destructure this?

```suggestion
                    {!isMobileViewLogin && <UrlNoticeBar />}
```